plugins/modules/eos_route_maps.py

```diff
@@ -0,0 +1,1404 @@
+                  choices: ["rename", "copy"]
+                source_map_name:
+                  description: Source route map name.
+                  type: str
+                overwrite:
+                  description: if True, overwrite existing config.
+                  type: bool
+            action:
+              description: Action for matching routes
+              type: str
+              choices: ["deny", "permit"]
+            sequence:
+              description: Index in the sequence.
+              type: int
+            continue_map:
+              description: Route map entry sequence number.
+              type: int
+            description:
+              description: Description for the route map.
+              type: str
+            sub_route_map:
+              description: Sub route map
+              type: dict
+              suboptions:
+                name:
+                  description: sub route map name
+                  type: str
+                invert_result:
+                  description: Invert sub route map result
+                  type: bool
+            set:
+              description: set route attributes.
+              type: dict
+              suboptions:
+                as_path:
+                  description: Set as-path.
+                  type: dict
+                  suboptions:
+                    match:
+                      description: Match the entire as-path.
+                      type: dict
+                      suboptions:
+                        as_number:
+                          description: as number to use (includes auto;in csv format)
+                          type: str
+                        none:
+                          description: Remove matching AS numbers
+                          type: bool
+                    prepend:
+                      description: Prepend to the as-path.
+                      type: dict
+                      suboptions:
+                        as_number:
+                          description: as number to prepend (includes auto;in csv format)
+                          type: str
+                        last_as:
+                          description: The number of times to prepend the last AS number.
+                          type: int
+                bgp:
+                  description: BGP AS path multipath weight.
+                  type: int
+                community_attributes:
+                  description: BGP community attribute.
+                  type: dict
+                  suboptions:
+                    graceful_shutdown:
+                      description: Graceful shutdown
+                      type: bool
+                    community:
+                      description: community attributes.
+                      type: dict
+                      suboptions:
+                        number:
+                          description: community number (in csv format).
+                          type: str
```

Comment:
community number should be list of number, correct?

Reply:
I am getting the list in csv format, which is processed inside the rm_template.